src/App.js

```diff
@@ -19,15 +23,21 @@ export class App extends Component {
               <div className="control">
                 <input
                   type="text"
+                  vcalue={this.state.query}
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  onChange={(e) => {
+                    this.setState({
+                      query: e.target.value.toLocaleLowerCase(),
```

Comment:
bad idea, do not limit the functionality available to the user if the user wants to write using upper case he should be able to do this, use toLowerCase behind the scene, so the user doesn't even notice those actions